controllers/os-suse-jeos/pkg/generator/generator.go

```diff
@@ -28,7 +30,15 @@ var cmd = "/usr/bin/cloud-init clean && /usr/bin/cloud-init --file %s init"
 // NewCloudInitGenerator creates a new Generator using the template file for suse-jeos
 func NewCloudInitGenerator() (*template_gen.CloudInitGenerator, error) {
 	box := packr.New("templates", "./templates")
-	cloudInitTemplateString, err := box.FindString("cloud-init-suse-jeos.template")
+
+	templateName, exists := os.LookupEnv("OS_CONFIG_FORMAT")
```

Comment:
please, put `OS_CONFIG_FORMAT` into a constant and document the acceptable values for the env var.